packages/extension-api/src/extension-api.d.ts

```diff
@@ -1858,6 +1858,61 @@ declare module '@podman-desktop/api' {
     export const clipboard: Clipboard;
   }
 
+  /**
+   * Options for running a command.
+   */
+  export interface RunOptions {
+    /**
+     * Environment variables to set for the command.
+     */
+    env?: { [key: string]: string };
+
+    /**
+     * A cancellation token used to request cancellation.
+     */
+    token?: CancellationToken;
+
+    /**
+     * A logger used to track execution events.
+     */
+    logger?: Logger;
+  }
+
+  /**
+   * Represents the result of running a command.
+   */
+  export interface RunResult {
+    /**
+     * The exit code of the command.
+     */
+    exitCode: number;
+
+    /**
+     * The standard output (stdout) content of the command.
+     */
+    stdout: string;
+
+    /**
+     * The standard error (stderr) content of the command.
+     */
+    stderr: string;
+  }
+
+  /**
+   * Namespace for environment-related utilities.
+   */
+  export namespace process {
+    /**
+     * Runs the provided command and returns an object containing the exit code,
+     * stdout, and stderr content. Supports cancellation using a CancellationToken.
+     * @param command The command to execute.
+     * @param args The command arguments.
+     * @param options Options, such as environment variables.
+     * @returns A promise that resolves to a RunResult object.
+     */
+    export function runCliCommand(command: string, args: string[], options?: RunOptions): Promise<RunResult>;
```

Comment:
what about just `run` or just `exec` ?

Reply:
maybe args should be optional as well

Reply:
agree, renamed the function name and made `args` parameter an optional